Apps/Contoso.UWP.Puppet/App.xaml.cs

```diff
@@ -27,6 +28,11 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
+            TaskScheduler.UnobservedTaskException += (object sender, UnobservedTaskExceptionEventArgs args) =>
+            {
+                // If you see this message while testing the app and if the stack trace is SDK related, we might have a bug in the SDK as we don't want to leak any exception from the SDK.
+                AppCenterLog.Error("AppCenterPuppet", "Unobserved exception observed=" + args.Observed, args.Exception);
```

Comment:
maybe report it to portal as handled exception?

Reply:
That would be nice, now it's out of scope for this work item.